src/components/structures/SpaceHierarchy.tsx

```diff
@@ -197,7 +197,7 @@ const Tile: React.FC<ITileProps> = ({
         );
     }
 
-    let description = _t("%(count)s members", { count: room.num_joined_members });
+    let description = _t("%(count)s members", { count: room.num_joined_members ?? 0 });
```

Comment:
Instead of displaying `0` what about just skipping the number of member information here?
Like we do for child rooms.